src/Chainweb/Chainweb/MinerResources.hs

```diff
@@ -86,14 +88,17 @@ withMiningCoordination logger conf cutDb inner
         t <- newTVarIO mempty
         c503 <- newIORef 0
         c403 <- newIORef 0
+        l <- newIORef (_coordinationUpdateStreamLimit conf)
```

Comment:
I thought you changed this back to 0?

Reply:
When this is zero all requests will be throttled. The initial value is the number of available streams. Requests borrow a unit from this counter and put it back when down.